src/web/components/ClientSideCompletion/ClientSideCompletion.tsx

```diff
@@ -0,0 +1,31 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import { Card } from '../Core/Card';
+import { KeyPairModel } from '../KeyPairs/KeyPairModel';
+
+import './ClientSideCompletion.scss';
+
+type ClientSideCompletionProps = Readonly<{
+  keyPairData: KeyPairModel[] | undefined;
+  domainNames: string[] | undefined;
+}>;
+export function ClientSideCompletion({ keyPairData, domainNames }: ClientSideCompletionProps) {
+  const hasKeyPairData = !!keyPairData?.filter((kp) => !kp.disabled).length;
```

Comment:
Should not have to filter on disabled here, looks like `keyPairData` only contains key pairs where disabled = true

Reply:
I'm going to leave this on the off-chance that we change the API to bring back disabled keys in the future.  It's probably best that the component not need to worry about the details of how the API is returning data.